From 75861583181839623bc9a6ea63ed7932d18f4f57 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Thu, 16 Jul 2020 11:19:03 -0400
Subject: [PATCH] spectemplate-python: Fix conditional for manual requires

---
 spectemplate-python.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spectemplate-python.spec b/spectemplate-python.spec
index 08ecffe..3625ca1 100644
--- a/spectemplate-python.spec
+++ b/spectemplate-python.spec
@@ -25,7 +25,7 @@ BuildRequires:  python%{python3_pkgversion}-setuptools
 Summary:        %{summary}
 %{?python_provide:%python_provide python3-%{pypi_name}}
 
-%if %{defined python_enable_dependency_generator} || %{defined python_disable_dependency_generator}
+%if %{undefined python_enable_dependency_generator} && %{undefined python_disable_dependency_generator}
 # Put manual requires here:
 Requires:       python%{python3_pkgversion}-foo
 %endif
-- 
2.26.2

